src/index.tsx

```diff
@@ -236,15 +250,24 @@ class OutPortal<C extends Component<any>> extends React.PureComponent<OutPortalP
     render() {
         // Render a placeholder to the DOM, so we can get a reference into
         // our location in the DOM, and swap it out for the portaled node.
-        // A <div> placeholder works fine even for SVG.
-        return <div ref={this.placeholderNode} />;
+        const tagName = this.props.node.element.tagName;
+
+        // SVG tagName is lowercase and case sensitive, HTML is uppercase and case insensitive.
+        // React.createElement expects lowercase first letter to treat as non-component element.
+        // (Passing uppercase type won't break anything, but React warns otherwise:)
+        // https://github.com/facebook/react/blob/8039f1b2a05d00437cd29707761aeae098c80adc/CHANGELOG.md?plain=1#L1984
+        const type = this.props.node.elementType === ELEMENT_TYPE_HTML
+            ? tagName.toLowerCase()
+            : tagName;
+
+        return React.createElement(type, { ref: this.placeholderNode });
```

Comment:
I added an example with a table (`tr`). Because `span`/`div` can't be added to the DOM directly under a `tbody`, React would complain about the placeholder. So, I create the placeholder element type based on the node elementType. This seems to work fairly well, but wanted to highlight